filebeat/fileset/fileset.go

```diff
@@ -37,7 +37,6 @@ func New(
 	name string,
 	mcfg *ModuleConfig,
 	fcfg *FilesetConfig) (*Fileset, error) {
-
 	modulePath := filepath.Join(modulesPath, mcfg.Module)
```

Comment:
I think it's cases like this one when a blank line helps readability? Otherwise, there's no indentation hint between the function header and the block, making it hard to read, IMO.